src/bindings/python/setup.in.py

```diff
@@ -34,8 +34,8 @@ def has_ext_modules(self):
 setup(
     name='@PYPI_PACKAGE_NAME@',
     version='@libCellML_VERSION@@LIBCELLML_DEVELOPER_VERSION@',
-    author='H. Sorby',
-    author_email='h.sorby@auckland.ac.nz',
+    author='libCellML Developers',
```

Comment:
I would put `developers` rather than `Developers`, but that's probably just me. Ok, I know it's just me! :)